Source/DafnyCore/DafnyOptions.cs

```diff
@@ -720,7 +720,7 @@ public enum IncludesModes {
           }
 
         case "warningsAsErrors":
```

Comment:
The release notes should mention that `/warningsAsErrors` is changing behavior too.